### ISSUE TITLE
Resume training with optimizer momentum

### ISSUE THREAD

**Issue by lesliecasas** [AlexeyAB/darknet#6720]
Is there anyway to save not only the weights but also the optimizer momentun in order to resume training?
It looks like there is only a save_weigths function.

**Comment by hoangtrung1999** [comment]
I'm facing the same problem too. The loss after I resume training becomes really high because the optimizer weights are not saved at all. Does anyone have solutions for this?

**Comment by ywssng** [comment]
@lesliecasas Is there a reason to save optimizer's momentum? Isn't the momentum and weight decay fixed?
